BibBuddy/app/src/main/java/de/bibbuddy/NoteModel.java

```diff
@@ -47,8 +64,43 @@ public Note getLastNote() {
    */
   public List<NoteItem> getCompleteNoteList() {
     List<Note> noteList = noteDao.findAll();
-    List<NoteItem> noteItemList = createItemList(noteList);
-    return noteItemList;
+    return createItemList(noteList);
+  }
+
+  /**
+   * This method gets the list of all existing notes of the voice-type (int=1).
+   *
+   * @return returns the list of voice type note objects.
+   */
+  public List<Note> getVoiceNoteList() {
+    List<Note> noteList = noteDao.findAll();
+    for (int i = 0; i < noteList.size(); i++) {
+      if (noteList.get(i).getType() != 1) {
+        noteList.remove(noteList.get(i));
+      }
+    }
+    return noteList;
+  }
+
+  /**
+   * This method creates a string representing the given date in a common readable format.
+   *
+   * @param date given date of type long representing millis since the start date in 1970.
+   * @return returns a string in "dd.mm.yyyy hh:mm:ss"-format.
+   */
+  public String getDate(Long date) {
+    Date d = new Date(date);
+    SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX",
+        Locale.getDefault());
+
+    String string = simpleDateFormat.format(d);
+    String day = string.substring(8, 10);
+    String month = string.substring(5, 7);
+    String year = string.substring(0, 4);
+    String time = string.substring(11, 16);
+    string = day + "." + month + "." + year + " " + time + " Uhr";
```

Comment:
Use noteItem.getModDateString with the adapter (line 134 in NotesRecyclerViewAdapter).

Reply:
Thanks - using the new suggested method now.